optimum/habana/diffusers/pipelines/stable_video_diffusion/pipeline_stable_video_diffusion.py

```diff
@@ -372,7 +383,7 @@ def __call__(
             # 4. Encode input image using VAE
             image = self.video_processor.preprocess(image, height=height, width=width)
             # torch.randn is broken on HPU so running it on CPU
-            rand_device = "cpu" if device.type == "hpu" else device
+            rand_device = torch.device("cpu") if device.type == "hpu" else device
```

Comment:
@sywangyi  is it still requiring rand to be on cpu to fix the black output issue?

Reply:
no, black issue is fixed by using fused sdpa, and sdp_on_bf16.
this "cpu“ is fix the ci failure.